apidoc/Titanium/UI/ShortcutItem.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: Titanium.UI.ShortcutItem
+summary: An application shortcut.
+description: |
+    Allows the creation of application shortcuts, which can be detected using
+    the `shortcutitemclick` event from <Titanium.App>.
+
+    Use the <Titanium.UI.createShortcutItem> method to create a shortcut.
+
+extends: Titanium.Proxy
+since: "7.2.0"
+methods:
+  - name: show
+    summary: Allow the shortcut to show.
+    platforms: [android, iphone, ipad]
+
+  - name: hide
+    summary: Hide the shortcut.
+    platforms: [android, iphone, ipad]
+
+properties:
+  - name: id
+    summary: Determines shortcut id.
+    description: This should be a unique identifier for the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: title
+    description: Title of the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: subtitle
+    description: Subtitle of the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: icon
+    summary: Shortcut icon.
+    description: This can be set as a resource id or file path.
+    type: [String, Number]
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+examples:
+  - title: Simple ShortcutItem Example (Android)
+    example: |
+        Create a shortcut item and detect its click with the `shortcutitemclick` event.
+
+            var win = Ti.UI.createWindow({ backgroundColor: 'grey' }),
+                btn = Ti.UI.createButton({ title: 'TOGGLE SHORTCUT' }),
+                shortcut = Ti.UI.createShortcutItem({
+                    id: 'test_shortcut',
+                    title: 'TEST',
+                    text: 'DESCRIPTION'
+                });
+
+            Ti.App.addEventListener('shortcutitemclick', function() {
+                win.backgroundColor = 'blue';
+            });
+
+            btn.addEventListener('click', function() {
+                if (shortcut.visible) {
+                    win.backgroundColor = 'red';
+                    shortcut.hide();
+                } else {
+                    win.backgroundColor = 'green';
+                    shortcut.show();
+                }
+            });
+
+            win.add(btn);
+            win.open();
```

Comment:
This does not play in parity with iOS, as iOS is not able to show/hide shortcuts. I updated the example to be Android only.